core/templates/pages/collection-editor-page/services/collection-editor-state.service.ts

```diff
@@ -77,6 +77,7 @@ angular.module('oppia').factory('CollectionEditorStateService', [
           collectionId).then(
           function(newBackendCollectionObject) {
             _updateCollection(newBackendCollectionObject);
+            $rootScope.$applyAsync();
```

Comment:
are we expecting this to be a quick fix and need to investigate and fix it later? If so can you please add a TODO? (maybe you can add the issue #8521 (not sure!))

Reply:
Hm, I think this is expected to be present. See 6(c) [here](https://github.com/oppia/oppia/wiki/Angular-Migration#implementation-details-to-migrate-services).

Reply:
In the link provided in that section, I see a `TODO` comment.

Reply:
Done.